app/controllers/user_controller.rb

```diff
@@ -0,0 +1,51 @@
+ServiceDiscovery::App.controllers :users do
+  
+  post '/:user_id/address' do
+    get_create_address_params
+    address = create_address @input, params[:user_id]
+    status 201
+    {address_id: address.id}.to_json
+  end
+
+  get '/:user_id/address' do
+    addresses = Address.where('user_id' => params[:user_id])
+    status 201
+    {addresses: addresses}.to_json
+  end
+
+  put '/:user_id/update_address' do
+    get_update_address_params
+    user_id = get_valid_user_id params[:user_id]
```

Comment:
Here is I am checking whether the user is valid or not and returning user_id. thats why I wrote the method as get_valid_user_id